lib/ansible/module_utils/vmware.py

```diff
@@ -557,6 +557,7 @@ def connect_to_api(module, disconnect_atexit=True, return_si=False):
         atexit.register(connect.Disconnect, service_instance)
     if return_si:
         return service_instance, service_instance.RetrieveContent()
+
```

Comment:
Nitpicking, could you reverse this change, this way your patch won't touch `lib/ansible/module_utils/vmware.py` at all.

Reply:
Done